### PR TITLE
[Translation] Add note about how to override translation in chi…

### PR DESCRIPTION
**Pull request by zebba**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets | #5193 |

### CODE REVIEW

File: cookbook/bundles/override.rst [hunk 1]
```diff
@@ -191,7 +191,10 @@ can override the translations from any translation file, as long as it is in
     The last translation file always wins. That means that you need to make
     sure that the bundle containing *your* translations is loaded after any
     bundle whose translations you're overriding. This is done in ``AppKernel``.
-
+    Translation files are not aware of the inheritance tree and therefore 
+    unaware of their parent. If you want to override translations from the 
+    parent bundle be sure the loading order in ``AppKernel`` is parent > child.
+
```
Comment (xabbuh) [inline cookbook/bundles/override.rst]: please remove the trailing whitespaces

File: cookbook/bundles/override.rst [hunk 2]
```diff
@@ -191,7 +191,10 @@ can override the translations from any translation file, as long as it is in
     The last translation file always wins. That means that you need to make
     sure that the bundle containing *your* translations is loaded after any
     bundle whose translations you're overriding. This is done in ``AppKernel``.
-
+    Translation files are not aware of the inheritance tree and therefore 
+    unaware of their parent. If you want to override translations from the 
+    parent bundle be sure the loading order in ``AppKernel`` is parent > child.
```
Comment (xabbuh) [inline cookbook/bundles/override.rst]: [...] bundle, be sure [...]

Comment (xabbuh) [inline cookbook/bundles/override.rst]: and I think we should put the "parent > child" part in words like:

> [...] sure that the parent bundle is loaded before the child bundle in the `AppKernel` class.

**Comment by xabbuh** [comment]
Thank you for starting this @zebba. I just left you a few small comments.

**Comment by zebba** [comment]
Ping @xabbuh

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks Sebastian! Easy merge :). Small tweak at sha:5467a56b3cd3298ca0b40757858c2a481df265e7 to link people to the cookbook on parent bundles, in case they don't know what we're talking about.

Cheers!

**Pull request merged**